docs/dev/table/connectors/kafka.md

```diff
@@ -0,0 +1,175 @@
+        <th class="text-center" style="width: 7%">Default</th>
+        <th class="text-center" style="width: 10%">Type</th>
+        <th class="text-center" style="width: 50%">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td><h5>connector</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Specify what connector to use, for Kafka the options are: 'kafka', 'kafka-0.11', 'kafka-0.10'.</td>
+    </tr>
+    <tr>
+      <td><h5>topic</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Required topic name from which the table is read.</td>
+    </tr>
+    <tr>
+      <td><h5>properties.bootstrap.servers</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Required Kafka server connection string.</td>
+    </tr>
+    <tr>
+    <td><h5>format</h5></td>
+    <td>required</td>
+    <td style="word-wrap: break-word;">(none)</td>
+    <td>String</td>
+    <td>Kafka connector requires to specify a format,
+    the supported formats are 'csv', 'json' and 'avro'.
+    Please refer to [Table Formats]({{ site.baseurl }}/dev/table/connect.html#table-formats) section for more details.
+    </td>
+        </tr>
+    <tr>
+      <td><h5>properties.group.id</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Required consumer group in Kafka consumer, no need for Kafka producer</td>
+    </tr>
+    <tr>
+      <td><h5>scan.startup.mode</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">group-offsets</td>
+      <td>String</td>
+      <td>Optional startup mode for Kafka consumer, valid enumerations are 'earliest-offset', 'latest-offset', 'group-offsets', 'timestamp' or 'specific-offsets'.</td>
+    </tr>
+    <tr>
+      <td><h5>scan.startup.specific-offsets</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Optional offsets used in case of 'specific-offsets' startup mode.
+      </td>
+    </tr>
+    <tr>
+      <td><h5>scan.startup.timestamp-millis</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>Long</td>
+      <td>Optional timestamp used in case of 'timestamp' startup mode.</td>
+    </tr>
+    <tr>
+      <td><h5>sink.partitioner</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Optional output partitioning from Flink's partitions into Kafka's partitions. Valid enumerations are
+      'fixed': (each Flink partition ends up in at most one Kafka partition),
+      'round-robin': (a Flink partition is distributed to Kafka partitions round-robin)
+      'custom class name': (use a custom FlinkKafkaPartitioner subclass).
```

Comment:
Use list for the accepted values, take https://ci.apache.org/projects/flink/flink-docs-master/ops/config.html#restart-strategy as an example.